README.md

```diff
@@ -91,6 +91,7 @@ For more details, please try [Getting started](./docs/en/getting-started.md).
 - Node.js environment 22.13.*
 - [`gcloud` CLI](https://cloud.google.com/sdk/docs/install)
 - [`jq` command](https://jqlang.org/)
+- [ `vale` command](https://github.com/errata-ai/vale)
```

Comment:
`Vale` is a dependency of development, but not just for running from the code.
Please move these instruction to [the development guide](https://github.com/GoogleCloudPlatform/khi/blob/main/docs/en/development-guide.md)